applications/MeshingApplication/python_scripts/mmg_process.py

```diff
@@ -155,6 +155,10 @@ def __init__(self, Model, settings ):
             "save_external_files"              : false,
             "save_colors_files"                : false,
             "save_mdpa_file"                   : false,
+            "remesh_at_finalize"               : true,
+            "output_final_mesh"                : true,
```

Comment:
IMO this should be set to `false` by default.

Reply:
I agree, this is important @federicoparisi